src/shipit_static_analysis/shipit_static_analysis/__init__.py

```diff
@@ -23,6 +25,20 @@ class Issue(object):
     - line: Line where the issue begins
     - nb_lines: Number of lines affected by the issue
     '''
+    def __init__(self):
+        # Read issue related content here to build an hash
+        full_path = os.path.join(settings.repo_dir, self.path)
+        assert os.path.exists(full_path), \
+            'Missing file {}'.format(full_path)
+
+        # Only read necessary lines
+        with open(full_path) as source:
+            lines = itertools.islice(source, self.line, self.line + self.nb_lines)
+            content = '\n'.join(lines)
+
+        # Build the content hash
+        self.lines_hash = hashlib.sha256(content.encode('utf-8')).hexdigest()
```

Comment:
The only problem I see with this is when the patch changes the indentation of the lines. We probably don't want to show the issue in that case, as it's basically equivalent to the case where the patch moves the lines without changing their indentation.

Reply:
So i just need to skip the leading space characteres of every lines. Sounds doable.